src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -30,83 +42,107 @@ public function __construct($config, $logCallback = null)
      */
     public function createTestUser($name, array $permissions)
     {
+        $appToken = $this->appId . '|' . $this->appSecret;
         $response = $this->executeFacebookRequest(
-            $this->appSession,
             'POST',
-            '/' . FacebookSession::_getTargetAppId() . '/accounts/test-users',
+            $this->appId . '/accounts/test-users',
+            $appToken,
             [
                 'name' => $name,
                 'installed' => true,
                 'permissions' => $permissions
             ]
-        )->getRawResponse();
+        );
 
-        return json_decode($response, true);
+        return $response->getDecodedBody();
     }
 
     public function deleteTestUser($testUserID)
     {
+        $appToken = $this->appId . '|' . $this->appSecret;
         $this->executeFacebookRequest(
-            $this->appSession,
             'DELETE',
-            '/' . $testUserID
+            '/' . $testUserID,
+            $appToken
         );
     }
 
     public function getTestUserInfo($testUserAccessToken)
     {
         $response = $this->executeFacebookRequest(
-            new FacebookSession($testUserAccessToken),
             'GET',
-            '/me'
-        )->getRawResponse();
+            '/me',
+            $testUserAccessToken
+        );
 
-        return json_decode($response, true);
+        return $response->getDecodedBody();
```

Comment:
yeah, but someone may have been using the other data, so if it get dissapearead it will break the test. @DavertMik what do you suggest ? should we introduce BC breack or should we try have it as it was before ?